# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/nsGUIEventIPC.h
# Commit: 88cd68e70502
# Full Hash: 88cd68e7050272bf76268c904feef7348d257b56
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-02-08 17:32:27
# Regressor Bug: 1746104
# File Overlap Count: 1
# Description:
#   Bug 1746104 - part 6-2: Make constructors of `ContentCache::Selection` take `IMENotification::SelectionChangeData` or `WidgetQueryContentEvent` r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137428
# ==============================================================================

diff -r 2529c39ed6bf -r 88cd68e70502 widget/nsGUIEventIPC.h
--- a/widget/nsGUIEventIPC.h	Mon Feb 07 22:33:39 2022 +0000
+++ b/widget/nsGUIEventIPC.h	Mon Feb 07 22:33:40 2022 +0000
@@ -939,6 +939,7 @@
     WriteParam(aMsg, aParam.mAnchor);
     WriteParam(aMsg, aParam.mFocus);
     WriteParam(aMsg, aParam.mWritingMode);
+    WriteParam(aMsg, aParam.mHasRange);
     WriteParam(aMsg, aParam.mAnchorCharRects[0]);
     WriteParam(aMsg, aParam.mAnchorCharRects[1]);
     WriteParam(aMsg, aParam.mFocusCharRects[0]);
@@ -951,6 +952,7 @@
     return ReadParam(aMsg, aIter, &aResult->mAnchor) &&
            ReadParam(aMsg, aIter, &aResult->mFocus) &&
            ReadParam(aMsg, aIter, &aResult->mWritingMode) &&
+           ReadParam(aMsg, aIter, &aResult->mHasRange) &&
            ReadParam(aMsg, aIter, &aResult->mAnchorCharRects[0]) &&
            ReadParam(aMsg, aIter, &aResult->mAnchorCharRects[1]) &&
            ReadParam(aMsg, aIter, &aResult->mFocusCharRects[0]) &&
